src/stc/stc.cpp

```diff
@@ -5062,7 +5062,19 @@ void wxStyledTextCtrl::AutoCompUseListCtrl(bool useListCtrl,
                                     currentTextColour);
 }
 
-
+void wxStyledTextCtrl::AutoCompShow(int lenEntered, const wxVector<wxCompletionItem>& list)
+{
+    m_autocompList.clear();
+    m_autocompList = list;
+    //Create correct format for Scintilla
+    wxString itemListString;
+    for(int i=0; i<m_autocompList.size(); i++)
```

Comment:
```suggestion
    for ( int i = 0; i < m_autocompList.size(); i++ )
```
please try to use style consistent with the rest of wx

Reply:
Thank you